Lesson_14/homework_14.py

```diff
@@ -0,0 +1,108 @@
+"""Create a class called "Student".
+
+Create a class called "Student" with attributes like
+"first name", "last name", "age", and "average grade".
+Create an object of this class to represent a student.
+Then add a method to the "Student" class
+that allows you to change the student's grade point average.
+Print out information about the student and change their grade point average.
+"""
+from logger import configure_logging
+
+_log = configure_logging()
+
+
+class Student:
+    """Class to represent a student's info."""
+
+    def __init__(self, first_name, last_name, age, average_grade):
+        """Initialize the student's attributes.
+
+        Args:
+            first_name (str): Student's first name.
+            last_name (str): Student's last name.
+            age (int): Student's age.
+            average_grade (float): Student's average grade.
+        """
+        self.first_name = first_name
+        self.last_name = last_name
+        self.age = age
+        self.average_grade = average_grade
+
+    def change_grade(self, new_grade):
+        """Change student's average grade.
+
+        Args:
+            new_grade (float): New student's average grade.
+
+        Returns:
+            string: _description_
+        """
+        # B010 "Do not call setattr with a constant attribute value,
+        # it is not any safer than normal property access.""
+        # if we use setattr(self, 'average_grade', new_grade)
+        self.average_grade = new_grade
+        message = f'Average grade is changed to : {new_grade}'
+        return message
```

Comment:
No no - here better to return self.avarage_grade. No sense to return some message . Better just log/print this message here